test/game_test.exs

```diff
@@ -8,35 +8,42 @@ defmodule TicTacToe.GameTest do
              board: nil,
              current_player: nil,
              next_player: nil,
-             game_over: false
+             game_status: :active
            }
   end
 
   test "Game.play/1 prints the 'new game' message" do
-    assert capture_io([input: "1\n1\n2\n3\n4\n5\n6\n7\n8\n9"], fn ->
+    assert capture_io([input: "1\n2\n3\n4\n5\n6\n7\n8\n9"], fn ->
```

Comment:
This particular input seems to be repeated throughout multiple tests. What do you think about turning this into a module attribute?